introduction.md

```diff
@@ -13,3 +13,5 @@ My Deadly Skills:
 * PHP
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?